### PR TITLE
Share cluster in ExpressionTestSupport

### PR DESCRIPTION
**Pull request by viliam-durina**
Reduces the affected tests form 69 seconds to about 44 seconds on my machine.
Also makes the laptop more quiet during that.

### CODE REVIEW

File: hazelcast-sql/src/test/java/com/hazelcast/sql/impl/expression/ExpressionTestSupport.java [hunk 1]
```diff
@@ -67,27 +78,42 @@
...
-    protected void before0() {
-        // No-op
+    @After
+    public void supportAfter() {
+        if (member == null) {
+            return;
+        }
+        // after each test ditch all jobs and objects
+        List<Job> jobs = member.getJetInstance().getJobs();
+        SUPPORT_LOGGER.info("Ditching " + jobs.size() + " jobs in SimpleTestInClusterSupport.@After: "
```
Comment (gierlachg) [inline hazelcast-sql/src/test/java/com/hazelcast/sql/impl/expression/ExpressionTestSupport.java:99]:
```suggestion
        SUPPORT_LOGGER.info("Ditching " + jobs.size() + " jobs in ExpressionTestSupport.@After: "
```

**Review by gierlachg — APPROVED** [review]
It has no effect on my local.

**Comment by viliam-durina** [comment]
> It has no effect on my local.

The test in the EXTRACT functions (not merged yet) has like 50 test cases, each started and shut down a member. It alone ran for 22 seconds locally, it will go down to 3 seconds i guess.

**Comment by viliam-durina** [comment]
run-lab-run

**Review by Fly-Style — APPROVED** [review]
Awesome job! 
I tried `CastFunctionIntegrationTest` and your changes reduce test time from 75s to 4s.

**Comment by viliam-durina** [comment]
run-lab-run

**Comment by viliam-durina** [comment]
run-lab-run

**Comment by viliam-durina** [comment]
run-lab-run

**Pull request merged**